lib/fs/MiqFsUtil.rb

```diff
@@ -322,6 +285,34 @@ def doUpdate(cs)
     end
   end
 
+  def check_encrypt(cs, i, toFile)
+    #
+    # If the file path matches an encrypt RE and doesn't
+    # match a noencrypt RE, then encrypt the contents of
+    # the file before copying it to the collection.
+    #
+    if cs.encrypt && cs.encrypt.detect { |e| i =~ e }
+      if !cs.noencrypt || !cs.noencrypt.detect { |ne| i =~ ne }
+        compressFile(i, toFile)
+        next
+      end
+    end
+  end
+
+  def check_compress(cs, i, toFile)
+    #
+    # If the file path matches an compress RE and doesn't
+    # match a nocompress RE, then compress the contents of
+    # the file before copying it to the collection.
+    #
+    if cs.compress && cs.compress.detect { |e| i =~ e }
+      if !cs.nocompress || !cs.nocompress.detect { |ne| i =~ ne }
+        compressFile(i, toFile)
+        next
```

Comment:
This `next` no longer has a corresponding loop.

Reply:
Ditto